src/status_im/contexts/wallet/add_account/create_account/import_private_key/view.cljs

```diff
@@ -0,0 +1,132 @@
+        (case state
+          :scanning
+          {:type    :info
+           :icon    :i/scanning
+           :message :t/scanning-for-activity}
+
+          :inactive-address
+          {:type    :warning
+           :icon    :i/info
+           :message :t/this-account-has-no-activity}
+
+          :active-address
+          {:type    :success
+           :icon    :i/done
+           :message :t/this-address-has-activity}
+
+          :invalid-private-key
+          {:type    :error
+           :icon    :i/info
+           :message :t/invalid-private-key}
+
+          nil)]
+    (when props
+      [quo/info-message
+       (assoc props
+              :size  :default
+              :style style/indicator)
+       (i18n/label message)])))
+
+(defn view
+  []
+  (let [customization-color           (rf/sub [:profile/customization-color])
+        [input-value set-input-value] (rn/use-state "")
+        [flow-state set-flow-state]   (rn/use-state)
+        error?                        (= :invalid-private-key flow-state)]
+    [rn/view {:flex 1}
+     [floating-button-page/view
+      {:customization-color customization-color
+       :header              [quo/page-nav
+                             {:background :white
+                              :type       :no-title
+                              :icon-name  :i/close
+                              :on-press   #(rf/dispatch [:navigate-back])}]
+       :footer              [:<>
+                             (when-not (#{:active-address :inactive-address} flow-state)
+                               [quo/information-box
+                                {:type  :default
+                                 :icon  :i/info
+                                 :style style/info-box}
+                                (i18n/label :t/import-private-key-info)])
+                             [quo/button
+                              {:customization-color customization-color
+                               :disabled?           (or (empty? input-value) error?)
+                               :on-press            #()}
+                              (i18n/label :t/continue)]]}
+      [quo/page-top
+       {:container-style  style/page-top
+        :title            (i18n/label :t/import-private-key)
+        :description      :text
+        :description-text (i18n/label :t/enter-private-key)}]
+      [address-input
+       {:input-value     input-value
+        :set-input-value set-input-value
+        :set-flow-state  set-flow-state
+        :error?          error?}]
+      (when (#{:scanning :active-address :inactive-address} flow-state)
+        [rn/view {:style style/key-section}
+         [quo/section-label
+          {:section         (i18n/label :t/private-key-public-address)
+           :container-style style/section-label}]
+         ;; TODO Get real address
+         [rn/view {:style (style/public-address flow-state)}
+          [quo/text "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045"]]])
```

Comment:
small nit: maybe you can just pull out all of the temporary hardcoded data into variables, even if it's in this file, just so it's easier to ensure we clean it up properly.

Reply:
I have `TODO` marks for highlighting such things

Reply:
sounds good! 🙏